### PR TITLE
Added ability to do range requests by providing a limit and an offset

### PR DESCRIPTION
**Pull request by lapistano**

### CODE REVIEW

**Comment by narcoticfresh** [comment]
hi.. the merge would remove a new line i added today - please merge develop in your branch and change the pr accordingly..

**Comment by lapistano** [comment]
the branch was created some minutes ago and git assures me it is up-to-date ...

File: lib/Graviton/Rql/Queriable/MongoOdm.php [hunk 1]
```diff
@@ -49,18 +49,32 @@ class MongoOdm implements QueryInterface
...
      *
      * @param DocumentRepository $repository repository
+     * @param int                $limit      Amount of documents to be returned
+     * @param int                $skip       Starting point from which the set of documents shall be returned
      */
-    public function __construct(DocumentRepository $repository)
+    public function __construct(DocumentRepository $repository, $limit = 10, $skip = 0)
     {
         $this->repository = $repository;
-        $this->classMetadata = $repository->getClassMetadata();
```
Comment (narcoticfresh) [inline lib/Graviton/Rql/Queriable/MongoOdm.php:56]: this line should not be removed..

**Comment by narcoticfresh** [comment]
changed a lot today, especially in MongoOdm.php.. issue a git fetch origin; then merge develop and you should see some changes..

**Pull request merged**